entproto/cmd/protoc-gen-entgrpc/cast.go

```diff
@@ -26,6 +26,8 @@ func (g *serviceGenerator) castToProtoFunc(fld *entproto.FieldMappingDescriptor)
 	// TODO(rotemtam): don't wrap if the ent type == the pb type
 	pbd := fld.PbFieldDescriptor
 	switch pbd.GetType() {
+	case dpb.FieldDescriptorProto_TYPE_BOOL:
+		return "bool", nil
```

Comment:
Looks good. You can use `reflect.<T>.String()` to avoid these strings btw, but it's not really needed.